packages/webpack-extraction-plugin/__fixtures__/webpack/missing-calls/output.ts

```diff
@@ -0,0 +1 @@
+export const foo = '__styles';
```

Comment:
wouldn't it make more sense to have better regex ? like `__styles()`? 🤔

Reply:
Yes, but this loader targets `node_modules` and needs to process thousands files in real projects, so if will put a regex in the hot path I believe that it will be a bottleneck.

Reply:
perhaps some kind of sliding window algorithm might be able to do the same thing while still being faster than Regex, especially you know how the call starts and terminates 🤔

but I guess handling empty buckets is a better gate